client/rest/Jenkinsfile

```diff
@@ -1,7 +1,9 @@
 defaultCiPipeline {
 	platform = ['ubuntu']
-	gitHubId = 'Symbol-Github-app'
+	ciBuildDockerfile = 'javascript.Dockerfile'
+
 	packageId = 'client-rest'
+
 	publisher = 'docker'
-	docker_image_name = 'symbolplatform/symbol-rest'
+	// dockerImageName = 'symbolplatform/symbol-rest'
```

Comment:
What's the plan for publishing images?

Reply:
idk, i just noticed that wasn't being used (and it was snake_case), so commented it out for now